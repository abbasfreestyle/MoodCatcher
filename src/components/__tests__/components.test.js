import * as components from '..';

describe('Components.', () => {
  describe('Snapshots.', () => {
    test('should match', () => {
      expect(components).toMatchSnapshot();
    });
  });
});
