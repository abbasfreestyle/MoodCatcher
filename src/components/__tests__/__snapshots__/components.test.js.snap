// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components. Snapshots. should match 1`] = `
Object {
  "Background": [Function],
  "Button": Object {
    "Card": [Function],
    "Feeling": [Function],
    "Regular": [Function],
  },
  "Chip": [Function],
  "Face": [Function],
  "Grid": Object {
    "Column": [Function],
    "Row": [Function],
  },
  "Item": Object {
    "Activity": [Function],
  },
  "Loading": [Function],
  "Title": [Function],
  "Touchable": [Function],
}
`;
